reactNativeProject/src/screens/Main/components/TodoList/components/checkbox/index.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { View, TouchableOpacity } from "react-native";
+import PropTypes from "prop-types";
+import styles from "./styles";
+
+function Checkbox(props) {
+  const { onToggle, isChecked } = props;
```

Comment:
When you define a functional component, you can make destructuring in the function declaration.